### ISSUE TITLE
Related model fields translation form missing

### ISSUE THREAD

**Issue by ghost** [zmei-framework/generator#2]
There are no form for translating fields of related model in admin panel.
![image](https://user-images.githubusercontent.com/41049901/53517219-cc821400-3ad6-11e9-9a0d-5f7587f65721.png)
/no translation tabs for awnser/
```
@suit
@langs(en, ru, et)

#question
--------
$=text: text
image: image
*~is_active: bool(false)

@admin(
list: *
inline: awnsers(
    type: stacked
)
)


#answer
-------
*question: one(!#question -> awnsers)
$=text: str
image: image
*~is_correct: bool(false)
```

**Comment by alex-rudakov** [comment]
@all-contributors please add @rumjantsevv for bug and userTesting

**Comment by alex-rudakov** [comment]
Merged into version 2.1.9.
